main.go

```diff
@@ -1,5 +1,59 @@
 package main
 
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/config"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/db"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/forman"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/reconciler"
+	"go.uber.org/zap"
+)
 
 func main() {
+	//logger := logr.Logger{}
+	ctx := context.Background()
+	configFilePath := os.Getenv("CONFIG_FILE_PATH")
+	if configFilePath == "" {
+		log.Fatal("CONFIG_FILE_PATH environment variable not set")
+	}
+	// parse configuration from yaml file
+	conf, err := config.LoadAndValidateConfig(configFilePath)
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
Why do we need a new package for config? can we just implement this in `config.go` on the main package?

Reply:
Do you suggest to pass the db details as input parameters not as struct to the DB Initialization?

Reply:
AFAIK we cannot have two files with package main, instead of a new package moved this functions to main.go itself